### ISSUE TITLE
why there's no banner rate up and pity rate system?

### ISSUE THREAD

**Issue by asudog1923** [ookamiiixd/ak-gacha#2]
So i'm just curious, will this project has other feature than only a super flat gacha?

**Comment by ookamiiixd** [comment]
Yes, there will be...my aim is to make this project look and work like the real one, so currently i'm still learning and gonna improved this project while learning.

**Comment by ookamiiixd** [comment]
Btw i'll lock this pull while improving the project and will close this when the target is reached.

**Comment by ookamiiixd** [comment]
You know what... i've completed the banner and pity rate system, just some final touch (gonna do it tomorrow since i'm so tired atm) and it will ready, maybe gonna update it in 2 or 3 more days or maybe faster depend on my condition.

**Comment by ookamiiixd** [comment]
I'm here to inform that i can't bring the update maybe atleast for a month because i've another project that i must set priority on it, so i;m really sorry.
